docusaurus/docs/overview/feature-model/scoping-rules.mdx

```diff
@@ -1,3 +1,8 @@
+---
+title: Scoping Rules
```

Comment:
We maybe should change the title to "Scoping *and* rules"...

Technically, *scoping* and *rules* are two different concepts.

1. All rule-based properties involve scoping.
1. But a property can be scoped, without being rule-based.